prime-router/metadata/schemas/covid-19.schema

```diff
@@ -180,6 +194,13 @@ elements:
     hl7Field: AOE
     hl7AOEQuestion: 65222-2
 
+  - name: observation_result_status
+    type: CODE
```

Comment:
I presume you are aware we already have two status fields, test_result_status and order_result_status.  
Two notes: 
1.  SimpleReport only uses test_result_status, and will soon be using it to mark data that is a correction of prior results.    We had some discussion about whether hl70123 or hl70085 was the correct value set to use.  They seemed absurdly similar, so I decided we could just map from one to the other in the Hub, as needed.

@MauriceReeves-usds , can you doublecheck my comment here: https://github.com/CDCgov/prime-data-hub/issues/344#issuecomment-763143528

to see if this makes you want to change over to hl70123?  Or, fell free to change test_result_status over to hl70085 if that's the right answer.

The current plan is for SimpleReport to only use F for Final, C for Corrected, and X to cancel a test result.  (See slack discussion here:  https://usds.slack.com/archives/C01H78NGM45/p1611087869002300

2.  Just for your awareness, Redox in all its glory has **three** _status fields:

```
Orders[].ResultStatus
Orders[].Status
Orders[].Results[].Status
```
each of which has slightly different sets of allowed values.  You are in a maze of little twisty passages, all alike....

Reply:
In some instances hl70123 and hl70085 do seem similar but hl70123 is used in the OBR section, and hl70085 is used in the OBX section. Some states are requesting we send them both OBR and OBX, for example North Dakota. I'm going to send this over to SimpleReport and ask them to review.

Reply:
@mauricereeves , I think they are so similar that we should just generate one from the other in the Hub.